neptune/exceptions.py

```diff
@@ -71,32 +112,132 @@ def __init__(self, expected_type, actual_type):
-                                                          .format(envs.PROJECT_ENV_NAME))
+        message = """
+{h1}
+----NeptuneMissingApiTokenException-------------------------------------------------------------------------------------
+{end}
+Neptune client couldn't find your API token.
+
+Learn how to get it in this docs page:
+https://docs.neptune.ai/security-and-privacy/api-tokens/how-to-find-and-set-neptune-api-token.html
+
+There are two options two add it:
+    - specify it in your code 
+    - set an environment variable in your operating system.
+
+{h2}CODE{end}
+Pass the token to {bold}neptune.init(){end} via {bold}api_token{end} argument:
+    {python}neptune.init(api_token='YOUR_LONG_TOKEN', project_qualified_name='WORKSPACE_NAME/PROJECT_NAME'){end}
+
+{h2}ENVIRONMENT VARIABLE{end}
+or export or set an environment variable depending on your operating system: 
+
+    {correct}Linux/Unix{end}
+    In your terminal run:
+        {bash}export {env_api_token}=YOUR_LONG_TOKEN{end}
+        
+    {correct}Windows{end}
+    In your CMD run:
+        {bash}set {env_api_token}=YOUR_LONG_TOKEN{end}
+        
+and skip the {bold}api_token{end} argument of {bold}neptune.init(){end}: 
+    {python}neptune.init(project_qualified_name='WORKSPACE_NAME/PROJECT_NAME'){end}
+    
+You may also want to check the following docs pages:
+    - https://docs.neptune.ai/security-and-privacy/api-tokens/how-to-find-and-set-neptune-api-token.html
+    - https://docs.neptune.ai/getting-started/quick-starts/log_first_experiment.html
+
+{correct}Need help?{end}-> https://docs.neptune.ai/getting-started/getting-help.html
+""".format(**{'env_api_token': envs.API_TOKEN_ENV_NAME, **STYLES})
+        super(NeptuneMissingApiTokenException, self).__init__(message)
+
+
+class NeptuneMissingProjectQualifiedNameException(NeptuneException):
+    def __init__(self):
+        message = """
+{h1}
+----NeptuneMissingProjectQualifiedNameException-------------------------------------------------------------------------
+{end}
+Neptune client couldn't find your project name.
+
+There are two options two add it:
+    - specify it in your code 
+    - set an environment variable in your operating system.
+
+{h2}CODE{end}
+Pass it to {bold}neptune.init(){end} via {bold}project_qualified_name{end} argument:
+    {python}neptune.init(api_token='YOUR_LONG_TOKEN', project_qualified_name='WORKSPACE_NAME/PROJECT_NAME'){end}
+
+{h2}ENVIRONMENT VARIABLE{end}
+or export or set an environment variable depending on your operating system: 
+
+    {correct}Linux/Unix{end}
+    In your terminal run:
+       {bash}export {env_project}=WORKSPACE_NAME/PROJECT_NAME{end}
 
+    {correct}Windows{end}
+    In your CMD run:
+       {bash}set {env_project}=WORKSPACE_NAME/PROJECT_NAME{end}
 
-class IncorrectProjectQualifiedName(NeptuneException):
+and skip the {bold}project_qualified_name{end} argument of {bold}neptune.init(){end}: 
```

Comment:
I'd surround the given project name with `"` in case it's empty, or a whitespace.

Reply:
Done